RoadMap_Instagram/RoadMap_Instagram/HomeScreenTable/Post.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Post.swift
+//  RoadMap_Instagram
+//
+//  Created by Артур Щукин on 17.10.2022.
+//
+
+import Foundation
+import UIKit
+
+/// ТЕст
+class Post {
+    var image: UIImage
+    var description: String
+    init(image: UIImage, description: String) {
```

Comment:
```suggestion
    init(image: UIImage, description: String) {
```
Init нужен в структуре?